FULL_HELP_DOCS.md

```diff
@@ -116,8 +116,6 @@ Get Id of builtin Soroban Asset Contract. Deprecated, use `stellar contract id a
 ###### **Options:**
 
 * `--asset <ASSET>` — ID of the Stellar classic asset to wrap, e.g. "USDC:G...5"
-* `--rpc-url <RPC_URL>` — RPC server endpoint
-* `--network-passphrase <NETWORK_PASSPHRASE>` — Network passphrase to sign the transaction sent to the rpc server
```

Comment:
Something I'm wondering about is folks might be using these in scripts, or with env vars, so that they can interact with specific RPC in a way that doesn't rely on stateful changes.

Reply:
fair point. I wonder if it's worth all the complexity on our codebase due to how command flattening happens with clap though. 🤔 

Any suggestions? Should we abandon this change?